.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,12 +1,13 @@
 name: Bug Report
 description: Use this template to report a bug or issue.
-title: "[Bug] "
-labels: ["bug", "needs review"]
+title: "[Question] "
+labels: ["question", "needs review"]
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report! You should only use this form for issues, to request a change or feature use the [feature request form](https://github.com/alexjustesen/speedtest-tracker).
+        Thanks for taking the time to fill out this bug report! You should only use this form for issues, to request a change or feature use the [feature request form](https://github.com/alexjustesen/speedtest-tracker/issues/new?template=feature_request.yml).
```

Comment:
We should update the wording to reflect something along the lines of thanking them for the issue and if it's confirmed as a bug the type will be change. Not quite sure how to word it best.